conf/config.go

```diff
@@ -222,11 +231,30 @@ func SaveConfigFile(config *MenderConfigFromFile, filename string) error {
 	return nil
 }
 
+func maybeHTTPSClient(c *MenderConfig) *client.HttpsClient {
+	if c.HttpsClient.Certificate != "" && c.HttpsClient.Key != "" {
+		return &c.HttpsClient
+	}
+	// TODO -- Duplicate of the above functionality - merge
```

Comment:
Some code duplication. I guess this is a question for @alfrunes 

Is the check and notification needed in the configuration code? For now it's in both places...